articles/app-service/overview-managed-identity.md

```diff
@@ -443,9 +443,19 @@ For .NET applications and functions, the simplest way to work with a managed ide
     // OR
     var kv = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(azureServiceTokenProvider.KeyVaultTokenCallback));
     ```
+If you want to use a user-defined managed identity, then you'll need to use the following code instead modifying to target the right resources and use the right clientid:
 
-To learn more about Microsoft.Azure.Services.AppAuthentication and the operations it exposes, see the [Microsoft.Azure.Services.AppAuthentication reference] and the [App Service and KeyVault with MSI .NET sample](https://github.com/Azure-Samples/app-service-msi-keyvault-dotnet).
+    ```csharp
+    using Microsoft.Azure.Services.AppAuthentication;
+    using Microsoft.Azure.KeyVault;
+    // ...
+    var azureServiceTokenProvider = new AzureServiceTokenProvider("RunAs=App;AppId=<clientid-guid>");
+    string accessToken = await azureServiceTokenProvider.GetAccessTokenAsync("https://vault.azure.net");
+    // OR
+    var kv = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(azureServiceTokenProvider.KeyVaultTokenCallback));
+    ```
```

Comment:
@cmatskas This one got lost with some of the other ones going in. Just the one more.

```suggestion
If you want to use a user-assigned managed identity, you can set the `AzureServicesAuthConnectionString` application setting to `RunAs=App;AppId=<clientid-guid>`, replacing "clientId-guid" with the client ID of the identity you wish to use. You can also define multiple such connection strings by using custom application settings and passing their values into the AzureServiceTokenProvider constructor.

    ```csharp
    var identityConnectionString1 = Environment.GetEnvironmentVariable("UA1_ConnectionString");
    var azureServiceTokenProvider1 = new AzureServiceTokenProvider(identityConnectionString1);
    
    var identityConnectionString2 = Environment.GetEnvironmentVariable("UA2_ConnectionString");
    var azureServiceTokenProvider2 = new AzureServiceTokenProvider(identityConnectionString2);
    ```
```

Reply:
I applied this revision.